go-controller/pkg/ovn/controller/services/utils.go

```diff
@@ -17,15 +18,18 @@ import (
 	"k8s.io/klog/v2"
 )
 
-// return the endpoints that belong to the IPFamily as a slice of IP:Port
-func getLbEndpoints(slices []*discovery.EndpointSlice, svcPort v1.ServicePort, family v1.IPFamily) []string {
+type lbEndpoints struct {
+	IPs  sets.String
```

Comment:
IIRC all the consumers of `lbEndpoints` use IPs.List(), I think it is better to make this directly a []string,  create a temporal sets.String{} to store the endpoint and, on return, add the list to `lbEndpoints`
something as:
```
	lbEps := lbEndpoints{[]string{}, 0}
        epsSet := sets.NewString()
        ....
         epsSet.Insert()
        ...
       lb.IPs = epsSet.List()
```

Reply:
ok will change